src/main/kotlin/com/terraformation/backend/eventlog/api/EventSubjectPayload.kt

```diff
@@ -59,6 +60,31 @@ sealed interface EventSubjectPayload {
   val shortText: String
 }
 
+@JsonTypeName("BiomassDetails")
+data class BiomassDetailsSubjectPayload(
+    override val fullText: String,
+    val monitoringPlotId: MonitoringPlotId,
+    val observationId: ObservationId,
+    val plantingSiteId: PlantingSiteId,
+    override val shortText: String,
+) : EventSubjectPayload {
+  companion object {
+    fun forEvent(
+        event: BiomassDetailsPersistentEvent,
+        context: EventLogPayloadContext,
+    ): BiomassDetailsSubjectPayload {
+      // TODO: Do we want something other than observation ID in the full text? Date or plot number?
+      return BiomassDetailsSubjectPayload(
```

Comment:
Are we leaving this TODO in here?

Reply:
Good catch. This will probably end up being more of a product question but I suspect we won't get a good answer until people can see this stuff in the UI and think about what they want it to look like.